be/src/runtime/snapshot_loader.cpp

```diff
@@ -655,7 +655,13 @@ Status SnapshotLoader::move(const std::string& snapshot_path, const TabletShared
         // link files one by one
         // files in snapshot dir will be moved in snapshot clean process
         std::vector<std::string> linked_files;
+        std::string dcg_file;
```

Comment:
It's better to describe the background, solution and effects in the commit message.